static/chat.delta.desktop.appdata.xml

```diff
@@ -0,0 +1,56 @@
+<!--<?xml version="1.0" encoding="UTF-8"?>-->
+<component type="desktop">
+  <id>chat.delta.desktop</id>
+  <metadata_license>CC0</metadata_license>
+  <project_license>GPLv3</project_license>
+  <name>Delta.chat</name>
+  <summary>Delta.Chat email-based messenger</summary>
+  
```

Comment:
Please change to just `Delta Chat`, without the dot (for \<name> and \<summary> that is)

Reply:
done